hil-test/Cargo.toml

```diff
@@ -72,7 +72,7 @@ embedded-hal       = "1.0.0"
 embedded-hal-02    = { version = "0.2.7", package = "embedded-hal", features = ["unproven"] }
 embedded-hal-async = { version = "1.0.0", optional = true }
 embedded-hal-nb    = { version = "1.0.0", optional = true }
-esp-backtrace      = { version = "0.11.1", git = "https://github.com/esp-rs/esp-backtrace", rev = "edff83c3945e8aa11081d8467af65803a82434ba", default-features = false, features = ["exception-handler", "panic-handler", "defmt", "semihosting"] }
+esp-backtrace      = { version = "0.11.1", path = "../esp-backtrace", default-features = false, features = ["exception-handler", "panic-handler", "defmt", "semihosting"] }
```

Comment:
Is there any reason to keep the `version`?

Reply:
The package is in the same repo, if the version doesn't match it won't build.